### PR TITLE
Add correct patterns

### PR DESCRIPTION
**Pull request by YuliiaKovalova**

### CODE REVIEW

File: src/DotUtils.MsBuild.SensitiveDataDetector/SensitiveDataDetectorFactory.cs [hunk 1]
```diff
@@ -6,37 +6,55 @@
 namespace Microsoft.Build.SensitiveDataDetector;
 
 [Flags]
-public enum SensitiveDataKind
+public enum SensitiveDataKind : long
```
Comment (JanKrivanek) [inline src/DotUtils.MsBuild.SensitiveDataDetector/SensitiveDataDetectorFactory.cs:9]: Btw. this won't scale up pretty soon.
We can then make this a normal enum and the requesting of the types to scan should not be possible to be that picky (probably just by categories of confidence, or maybe even not that - we should just allways scan everything. Only username feels as something to be possibly legit to opt-out)

File: src/DotUtils.MsBuild.SensitiveDataDetector/DotUtils.MsBuild.SensitiveDataDetector.csproj [hunk 2]
```diff
@@ -19,7 +19,7 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <LangVersion>preview</LangVersion>
```
Comment (JanKrivanek) [inline src/DotUtils.MsBuild.SensitiveDataDetector/DotUtils.MsBuild.SensitiveDataDetector.csproj:22]: Brave! :-)

**Review by JanKrivanek — APPROVED** [review]
Perfect! Thank you!!

File: src/DotUtils.MsBuild.SensitiveDataDetector/SensitiveDataDetectorFactory.cs (hunk 1 above)
Comment (YuliiaKovalova) [reply src/DotUtils.MsBuild.SensitiveDataDetector/SensitiveDataDetectorFactory.cs]: I reverted it to the initial state

**Pull request merged**